server/stycoon/game.go

```diff
@@ -2,20 +2,54 @@ package stycoon
 
 import (
 	"database/sql"
+	"fmt"
+	"strconv"
 	"time"
 )
 
 type Game struct {
-	Tick     CurrentTick
-	db       *sql.DB
-	lastTick time.Time
+	Tick          CurrentTick
+	db            *sql.DB
+	lastTick      time.Time
+	ResourceNames map[string]string
+	ShipClasses   map[string]StaticDataShipClassesValue
```

Comment:
what is the convention for naming variables in go? why do some start with upper case and some with lower case?

Reply:
Case determines visibility.
Uppercase is public.
Lowercase is private.